lib/components/T3CeFeloginLogin/T3CeFeloginLogin.vue

```diff
@@ -0,0 +1,80 @@
+<template>
+  <div class="fe-login">
+    <p>{{ $props.data.message.header }}</p>
```

Comment:
you can omit `$props` here, just start with data

Reply:
Done ✅